src/Files.App/ServicesImplementation/QuickAccessService.cs

```diff
@@ -53,8 +53,10 @@ public async Task UnpinFromSidebar(string[] folderPaths)
 
 			foreach (dynamic? fi in f2.Items())
 			{
+				App.Logger.Info(fi.Path);
 				if (folderPaths.Contains((string)fi.Path)
-					|| (string.Equals(fi.Path, "::{645FF040-5081-101B-9F08-00AA002F954E}") && folderPaths.Contains(CommonPaths.RecycleBinPath)))
+					|| (string.Equals(fi.Path, "::{645FF040-5081-101B-9F08-00AA002F954E}") && folderPaths.Contains(Constants.CommonPaths.RecycleBinPath))
+					|| (string.Equals(fi.Path, "::{F02C1A0D-BE21-4350-88B0-7367FC96EF3C}") && folderPaths.Contains(Constants.CommonPaths.NetworkFolderPath)))
```

Comment:
Any reason to not use the constants values in the `string.Equals`?

Reply:
We can make them into constants